src/scss/components/_carousel.scss

```diff
@@ -356,6 +356,14 @@
 		}
 	}
 
+	&__item {
+		&.carousel-item-prev.carousel-item-right {
+			@include media-breakpoint-down(md) {
```

Comment:
we discussed about this syntax 2 days ago. breakpoint should come before the classes, also if you don't have other nested element for &__item don't need to nest &:carousel.. just attach. Please careful with syntax

Reply:
@foria 
moved the breakpoint before the .carousel-item-prev.carousel-item-right

I've tried to attach .carousel-item-prev.carousel-item-right to &__item, but its not possible, because the breakpoint cannot be put as a parent of &__item